src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -1660,30 +1660,66 @@ subroutine li_apply_front_ablation_velocity(meshPool, geometryPool, velocityPool
       allocate(thicknessForAblation(nCells+1))
       thicknessForAblation = 0.0_RKIND
 
-
       ! 1. Calculate ablation rate for all non-dynamic cells by working with their ocean-going edges
       ablationSubtotal1 = 0.0_RKIND
       do iCell = 1, nCellsSolve
          if (li_mask_is_floating_ice(cellMask(iCell)) .and. (.not. li_mask_is_dynamic_ice(cellMask(iCell)))) then
-            ! a. Translate the calving front height based on dynamic cells to the non-dynamic locations
+            ! a1. Translate the calving front height based on dynamic cells to the non-dynamic locations
             ! find mean (or min?) of thickness in dynamic neighbors
+            ablationVelSum = 0.0_RKIND
             thkSum = 0.0_RKIND
             thkCount = 0
             do iNeighbor = 1, nEdgesOnCell(iCell)
                jCell = cellsOnCell(iNeighbor, iCell)
                if (li_mask_is_dynamic_ice(cellMask(jCell))) then
                   thkSum = thkSum + thickness(jCell)
+                  ablationVelSum = ablationVelSum + ablationVelocity(jCell)
                   thkCount = thkCount + 1
                endif
             enddo
             if (thkCount == 0) then
-               !call mpas_log_write("Found a stranded non-dynamic floating cell: cell $i with thickness=$r m.", MPAS_LOG_WARN, &
+                !call mpas_log_write("Found a stranded non-dynamic floating cell: cell $i with thickness=$r m.", MPAS_LOG_WARN, &
```

Comment:
picky comment: undo the indentation change here and in the `else` clause immediately following.

Reply:
Fixed